settings.gradle.kts

```diff
@@ -35,6 +35,13 @@ javaModules {
         module("swirlds-unit-tests/core/swirlds-platform-test") // nested module is not found automatically
     }
 
+    directory("consensus") {
+        group = "com.hedera.hashgraph"
+
+        module("consensus-gossip") { artifact = "app-consensus-gossip" }
```

Comment:
What is the artifact line doing? Does this mean the jar file will be called "app-consensus-gossip"?